### PR TITLE
remove jQuery or Zepto dependency

### PR DESCRIPTION
**Pull request by IegorKozakov**
* remove jQuery or Zepto dependency
* minify quickblox.min.js

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -26,13 +32,46 @@ gulp.task('build', function () {
...
+            '-offset',
+            '-wrap',
+            '-core/ready',
+            '-exports/global',
+            '-sizzle'
+        ],
+    }, function (err, compiledContent) {
+        if (err) return console.error(err);
+        fs.writeFile('./src/plugins/jquery.ajax.js', compiledContent, function (err) {
+            if (err) return console.error(err);
```
Comment (dimaspirit) [inline gulpfile.js]: Also use notify for notify module about a crash process.

**Comment by dimaspirit** [comment]
We are using npm commands. Like `npm run develop` or `npm run build`.
So add gulp compress to process by npm commands or include in gulp commands.

**Comment by soulfly** [comment]
plus +

File: gulpfile.js (hunk 1 above)
Comment (IegorKozakov) [reply gulpfile.js]: done

File: package.json [hunk 2]
```diff
@@ -74,7 +77,7 @@
   "scripts": {
     "setDependencies": "npm i && npm install -g gulp-cli && npm install -g jasmine",
     "lint": "jshint src --reporter=node_modules/jshint-stylish",
-    "build": "npm run lint && gulp generate-build_version && gulp build",
+    "build": "npm run lint && gulp generate-build_version && gulp build_min",
```
Comment (dimaspirit) [inline package.json]: You need to build and minify. Like this `"build": "npm run lint && gulp generate-build_version && gulp build && gulp minify"`

File: gulpfile.js [hunk 3]
```diff
@@ -57,8 +72,32 @@ gulp.task('generate-build_version', function() {
...
+        })
+    })
+});
+
 gulp.task('watch', function () {
   gulp.watch(['./src/**/*.js'], ['build']);
 });
 
-gulp.task('default', ['build', 'connect', 'watch']);
+gulp.task('default', ['build_sdk', 'connect', 'watch']);
```
Comment (dimaspirit) [inline gulpfile.js]: Where are you find the task `build_sdk`?

File: gulpfile.js (hunk 1 above)
Comment (dimaspirit) [reply gulpfile.js]: Are you sure?

**Pull request merged**